Makefile

```diff
@@ -58,7 +78,7 @@ endif
 
 # Bootstrapped by Operator-SDK v1.1.0
 # Current Operator version
-VERSION ?= 0.0.1
+DWO_VERSION ?= 0.0.1
```

Comment:
I'm not sure if we need it in Makefile

Reply:
I still think we should have the only one place to devfile semver for DWO and currently it's https://github.com/devfile/devworkspace-operator/blob/main/version/version.go#L16

Reply:
Sorry, missed this comment earlier. Okay, I've removed it!